xbmc/addons/settings/AddonSettings.cpp

```diff
@@ -316,7 +316,7 @@ bool CAddonSettings::InitializeFromOldSettingDefinitions(const CXBMCTinyXML& doc
     std::vector<SettingWithConditions> settingsWithConditions;
 
     // prepare a setting group
-    std::shared_ptr<CSettingGroup> group(new CSettingGroup("0", GetSettingsManager()));
+    auto group = std::make_shared<CSettingGroup>("0", GetSettingsManager());
```

Comment:
Hm? `std::make_shared<>()` should be available in C++11. It's only `std::make_unique<>()` which is missing in C++11 and has been added in C++14. Or did you mean something else?